packages/wallet-frontend/src/pages/account/create.tsx

```diff
@@ -56,7 +56,6 @@ export default function CreateAccount({ assets }: CreateAccountProps) {
       >
         <Input
           required
-          placeholder="My Account"
           label="Account name"
           error={createAccountForm.formState?.errors?.name?.message}
           {...createAccountForm.register('name')}
```

Comment:
Should add to the Select in this file a label (Asset or Currency) and a placeholder (Select asset or currency...)